bower.json

```diff
@@ -8,6 +8,7 @@
     "angular-datamaps": "^0.1.2",
     "angular-datatables": "0.5.6",
     "angular-nvd3": "^0.1.1",
+    "angular-nvd3-1.0.9": "angular-nvd3#1.0.9",
```

Comment:
It looks like this library is only used in icds_reports. is there a reason to have it separated like this

Reply:
The library was updated in hope of fixing tooltip bug which was getting frozen, but because of big differences in versions, changes to the code using this library has to be changed. So to keep the changes to the minimum this version is only used in icds_reports.

Reply:
@dmydlo Actually I meant that only icds_reports uses angular-nvd3, so it should be ok to upgrade that instead of having two versions

Reply:
Because of big differences in versions, changes to the code using this library would have to be made. In case of changing version of this library across the whole project, there would be changes in code across the entirety of the code which could lead to many bugs.